packages/apps/crm/src/components/organisms/Contact/ContactDropdownCards/ContactDropdownCards.js

```diff
@@ -71,6 +72,11 @@ const ContactDropdownCards = ({}) => {
             key: 20,
             childrenComp: <DropdownEventView eventList={listEventContact} />,
           },
+          {
+            title: I18n.t('Crm_Clients'),
```

Comment:
Please change to "Linked clients and/or prospects" / "Clients et/ou prospects associés"